src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+
+import codeu.chat.util.Uuid;
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  public ServerInfo(){
+      Uuid temp = null;
+      try {
+        temp = Uuid.parse(SERVER_VERSION);
```

Comment:
Just wanted to chime in here. A try/catch block is actually necessary here because the java compiler requires it. IOExceptions are a class of exceptions that we need to catch in our code, otherwise it just won't compile at all :)